### ISSUE TITLE
We should redact/hide provider creds in debug logs for registerFunction call

### ISSUE THREAD

**Issue by rupakg** [serverless/event-gateway#232]
The `awsAccessKeyID` and `awsSecretAccessKey` values should be redacted or hidden in the logs. 

```
DEBUG	Function registered.	{"functionId": "test", "type": "awslambda"}
DEBUG	Function cache received set key.	{"key": "test", "value": "{\"functionId\":\"test\",\"provider\":{\"type\":\"awslambda\",\"arn\":\"arn:aws:lambda:us-east-1:063733696545:function:user-service-prod-test\",\"region\":\"us-east-1\",\"awsAccessKeyID\":\"junk\",\"awsSecretAccessKey\":\"junk\"}}"}
```

**Comment by mthenw** [comment]
Good point @rupakg!

**Comment by alexdebrie** [comment]
This was closed in #385.
